woocommerce/woocommerce
# Add extra unique event names for CI jobs

### Changes proposed in this Pull Request:

Add new unique event names for each CI job. This will add great flexibility when running tests using the on demand workflow.

### How to test the changes in this Pull Request:

1. Go to [On demand](https://github.com/woocommerce/woocommerce/actions/workflows/tests-on-demand.yml) workflow
2. Select this branch, custom as the event name and fill in one of the new events added in this PR
3. Run the workflow and check the jobs that ran

Only the jobs matching the given event should run

See [this](https://github.com/woocommerce/woocommerce/actions/runs/12390672292/job/34586193083) test run.
